pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/FunctionWorkerStarter.java

```diff
@@ -65,13 +67,17 @@ public static void main(String[] args) throws Exception {
         }
 
         WorkerConfig workerConfig;
+        ServiceConfiguration serviceConfiguration;
         if (isBlank(workerArguments.configFile)) {
             workerConfig = new WorkerConfig();
+            serviceConfiguration = new ServiceConfiguration();
         } else {
             workerConfig = WorkerConfig.load(workerArguments.configFile);
+            serviceConfiguration = PulsarConfigurationLoader.create(workerArguments.configFile, ServiceConfiguration.class);
```

Comment:
This doesn't look correct. the `workerArguments.configFile` points to the `function_worker.yml` file. However PulsarConfigurationLoader.create expects a properties file. I guess this would make sense if PulsarConfigurationLoader was modified to support yaml files?

Reply:
You are correct. In my haste to get a solution, I missed the file extensions that should have warned me that I was misunderstanding how things worked. I'll close this PR. Thank you for this helpful comment.